internal/events/event.go

```diff
@@ -139,6 +189,16 @@ func (e *Events) AddMetricsEvent(ctx context.Context, clusterID strfmt.UUID, hos
 	_ = e.saveEvent(ctx, clusterID, hostID, models.EventCategoryMetrics, severity, msg, eventTime, requestID, props...)
 }
 
+func (e *Events) V2AddEvent(ctx context.Context, clusterID *strfmt.UUID, hostID *strfmt.UUID, infraEnvID *strfmt.UUID, severity string, msg string, eventTime time.Time, props ...interface{}) {
+	requestID := requestid.FromContext(ctx)
+	_ = e.v2SaveEvent(ctx, clusterID, hostID, infraEnvID, models.EventCategoryUser, severity, msg, eventTime, requestID, props...)
```

Comment:
Should we preserve the behavior of ignoring failed save events ?

Reply:
this is not ideal. indeed based on the existing `AddEvent` function from v1.
However, looking at the code I think we will need to make a lot of changes in the calling functions to have the error returned all the way up, which might be out of scope for this PR.

Should we just log the error under `V2AddEvent()` in this case?

Reply:
Logs sounds good. it gives some idea to the user that some expected event didn't happen due to an error.